db/migrate/20160324124129_add_index.rb

```diff
@@ -0,0 +1,4 @@
+class AddIndex < ActiveRecord::Migration
+  def change
```

Comment:
Rails does it's own validation rather then just relying on the db which I find really annoying for things like this.

If you delete the migration and revert the `schema.rb` I'll add the validation in another PR.